### DISCUSSION TITLE
Default links in login footer

### DISCUSSION THREAD

**Discussion by mffap** [zitadel/zitadel#2764] [Q&A] [answered]
Depending on how you use ZITADEL, you might want to set the default values differently.

At the moment global default values are shown on new organizations. This makes sense in certain use cases.

Other use cases might require to hide the links by default and only show them if the organization has set a value explicitly.

The 'Help' link should be configurable like the other two links, thus allowing to be set to the organization's/product's help page.

- [x] Make help link configurable
- [x] Allow to set no value for all links globally by default
- [x] Hide links if they are not set explicitly or through default values

**Comment by hifabienne** [comment, marked as answer]
This is solved with the privacy policy and tos settings. 
New help link is added and they all can be empty
